tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -110,7 +110,11 @@ ifneq ($(CO_PROCESSOR),)
   ADDITIONAL_DEFINES += -D$(shell echo $(CO_PROCESSOR) | tr [a-z] [A-Z])
 endif
 
-CORE_OPTIMIZATION_LEVEL := -Os
+ifeq ($(TOOLCHAIN), armclang)
+  CORE_OPTIMIZATION_LEVEL := -Oz
+else ifeq ($(TOOLCHAIN), gcc)
```

Comment:
Setting CORE_OPTIMIZATION_LEVEL to -Os should probably not be conditional on ifeq($(TOOLCHAIN), gcc). We use both gcc and clang, and having a catch-all here is probably best.

Reply:
I uploaded a mew commit, was it that you had in mind? I also just realized that I forgot to create and reference an issue. I'll edit my first comment with a link to it.